### ISSUE TITLE
[Q] File won't be saved properly

### ISSUE THREAD

**Issue by theMasix** [exceljs/exceljs#1286]
I wrote an script and here's an snippet from my code:
```
let workbook = new Excel.Workbook();
await workbook.xlsx.readFile('./input.xlsx');
await workbook.xlsx.writeFile('./output.xlsx');
console.log('file saved!');
```
I know how async/await works and there is no error!
the ```input.xlsx``` is a valid excel file. but the ```output.xlsx``` can't be opened by libreOffice or Microsoft Excel.
I should notice that if I comment ```await workbook.xlsx.readFile('./input.xlsx');```, in this situation also the ```output.xlsx``` can't be read too!

It's an strange issue for me! I used ```exceljs``` many many times but I don't know why it's happening now!

```exceljs``` version: ```3.9.0```

**Comment by Alanscut** [comment]
@theMasix Can you provide the `input.xlsx`? This is helpful for us to solve the problem 😄.

**Comment by theMasix** [comment]
@Alanscut It could be any ```xlsx``` file. I tested it with many files Including empty file even! The output couldn't be opened anyway.

**Comment by Alanscut** [comment]
@theMasix I use  the following code and  it works fine.

```javascript
const Excel = require('exceljs');

const wb = new Excel.Workbook();
async function create() {
  await wb.xlsx.readFile('./issue1287.xlsx');
  await wb.xlsx.writeFile('./output.xlsx');
}
create()
```

**Comment by theMasix** [comment]
@Alanscut Finally I figured out what the issue was!
All columns and rows were hidden and strangely in this situation, the excel couldn't be opened!
But if I add the visibility to each row and col, the file would open too!
So the problem is solved now.
But I don't know why all rows and cols were hidden after saving.

**Comment by Alanscut** [comment]
@theMasix Can you please provide the test code and the version of miscosoft excel, I did not reproduce this error.

**Comment by Siemienik** [comment]
Info: issue originally started on stack overflow: https://stackoverflow.com/questions/61922359/exceljs-doesnt-save-file-properly/61953353?noredirect=1#comment109626568_61953353

@Alanscut Exceljs saved some information in a different way than Excel. It's hard to follow it.
I think the most sensible solution to track these differences is unzip Input and Output files and make a diff between two folders.

BTW. I had a similar issue with images, fixed in #702

**Comment by theMasix** [comment]
@Alanscut The code is:
```let workbook = new Excel.Workbook();
await workbook.xlsx.readFile('./input.xlsx');
await workbook.xlsx.writeFile('./output.xlsx');
console.log('file saved!');
```
input file can be something like: [input.xlsx](https://gofile.io/d/Pujfrn)
I don't use Microsoft Excel. I use ```libreoffice calc``` version: ```6.4.3.2```

**Comment by theMasix** [comment]
@Siemienik @Alanscut  And the strange thing is that I used ```exceljs``` for a while and I don't have any issue with it. But now and on ```libreoffice calc``` this issue happens. I don't know if the issue is for ```libreoffice``` or not!
And I should point something else: If I use some online Excel viewer like [SheetJS In-Browser Live](https://oss.sheetjs.com), the file will open, but file won't open with ```libreoffice calc```
It's another strange thing :(
